public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -24,11 +24,29 @@ const INTERVAL_FACTOR_OPTIONS: Array<SelectableValue<number>> = map([1, 2, 3, 4,
   label: '1/' + value,
 }));
 
+const STEP_OPTIONS: Array<SelectableValue<StepType>> = [
+  {
+    value: 'min',
+    label: 'minimum',
```

Comment:
The first letter in label should be uppercased (e.g Minimum, Maximum, ...) to keep UI consistent.